hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterServiceImpl.java

```diff
@@ -400,7 +401,11 @@ public boolean finalizeJoin(MembersView membersView, Address callerAddress, UUID
             membershipManager.updateMembers(membersView);
             clusterHeartbeatManager.heartbeat();
             setJoined(true);
-
+            node.getNodeExtension().getAuditlogService()
+                .eventBuilder(AuditlogTypeIds.CLUSTER_MEMBER_ADDED)
+                .message("Member joined")
+                .addParameter("membersView", membersView)
```

Comment:
Do we want to log which member joined?

Reply:
:+1: I'll add it as new parameter: `address`